htdocs/core/lib/company.lib.php

```diff
@@ -710,51 +710,25 @@ function getFormeJuridiqueLabel($code)
  */
 function getCountriesInEEC()
 {
-	global $conf;
-
-	// List of all country codes that are in europe for european vat rules
-	// List found on http://ec.europa.eu/taxation_customs/common/faq/faq_1179_en.htm#9
-	$country_code_in_EEC = array(
-		'AT', // Austria
-		'BE', // Belgium
-		'BG', // Bulgaria
-		'CY', // Cyprus
-		'CZ', // Czech republic
-		'DE', // Germany
-		'DK', // Danemark
-		'EE', // Estonia
-		'ES', // Spain
-		'FI', // Finland
-		'FR', // France
-		'GB', // United Kingdom
-		'GR', // Greece
-		'HR', // Croatia
-		'NL', // Holland
-		'HU', // Hungary
-		'IE', // Ireland
-		'IM', // Isle of Man - Included in UK
-		'IT', // Italy
-		'LT', // Lithuania
-		'LU', // Luxembourg
-		'LV', // Latvia
-		'MC', // Monaco - Included in France
-		'MT', // Malta
-		//'NO',	// Norway
-		'PL', // Poland
-		'PT', // Portugal
-		'RO', // Romania
-		'SE', // Sweden
-		'SK', // Slovakia
-		'SI', // Slovenia
-		'UK', // United Kingdom
-		//'CH',	// Switzerland - No. Swizerland in not in EEC
-	);
+	global $conf, $db;
+	$country_code_in_EEC = array();
 
-	if (!empty($conf->global->MAIN_COUNTRIES_IN_EEC)) {
```

Comment:
For backward compatibility, i suggest to keep this test and use the constant instead of sql if it has been defined.